application_form/api/sales/serializers.py

```diff
@@ -46,22 +46,22 @@ class Meta:
 
 class SalesApartmentReservationSerializer(ApartmentReservationSerializerBase):
     applicants = ApplicantCompactSerializer(
-        source="application_apartment.application.applicants", many=True
+        source="application_apartment.application.applicants",
+        many=True,
+        allow_null=True,
```

Comment:
Hmm, why is `null` needed here? Shouldn't a non-existing value be an empty list instead of null?

Reply:
The `null` value is to handle the case that the application is `null`, i.e reservation created by sales person without application. Without `allow_null=True` that field will raise `AttributeError`. In any other case, it'll be an empty list if there is no applicant.

Reply:
Ok makes sense 👍